PyInstaller/hooks/hook-shelve.py

```diff
@@ -0,0 +1,3 @@
+# Hidden Imports for Shelve Module
```

Comment:
@Penaz91 Thans for your hook. Could you please add here the license header similar to other hooks and a comment about the shelve version and OS you tested it with?